example/aiur.config.js

```diff
@@ -10,3 +10,6 @@ module.exports = {
 		}
 	}
 };
+
+exports.title = "Example Styleguide";
+exports.language = "en";
```

Comment:
It's easy to miss this down here; I'd move these up top, i.e. above `pages`.

Reply:
You mean all three of them, right? I agree.

Reply:
Are you gaslighting me?